test/OrchardCore.Commerce.Tests/PriceTests.cs

```diff
@@ -107,7 +107,14 @@ public DummyPriceProvider(int priority, decimal price)
         public Task<IEnumerable<ShoppingCartItem>> AddPricesAsync(IList<ShoppingCartItem> items) =>
```

Comment:
Unit tests are good, but don't we need UI tests as well?

Reply:
This need to be discussed in the future too. But I talked about this with @barthamark and we think that because it's payment and it's using external services we can't UI test it. It's possible that we are wrong though.

Reply:
You can add a mock service for testing, turn it on if the side is running in UI test mode. Also you can have tests that work with an existing Order content items (the stuff you've added to Display and Edit works as expected) that can be added with recipe. Let's consider this technical debt for now, I will open a followup issue.

Reply:
Yeah, I was about to say that if it's possible handle it as a separate issue. Thanks!

Reply:
I've created the issue here: #130. If you aren't completely fed up with Stripe, please assign yourself to it but it's not a priority right now.